apps/ui/src/networks/offchain/actions.ts

```diff
@@ -428,9 +443,7 @@ export function createActions(
       });
     },
     deleteSpace: async (web3: Web3Provider, space: Space) => {
-      if (space.snapshot_chain_id) {
-        await verifyNetwork(web3, space.snapshot_chain_id);
-      }
+      await verifyNetwork(web3, chainId);
```

Comment:
I think we verify `deleteSpace` against `snapshot_chain_id`:
https://github.com/snapshot-labs/snapshot-sequencer/blob/63420e0a67a335a97b948677f2585fbafd4a036c/src/ingestor.ts#L94-L105

Reply:
hmm there is no relation to snapshot_chain_id for `deleteSpace` , need to update sequencer

Reply:
Created a PR https://github.com/snapshot-labs/snapshot-sequencer/pull/507